junit5/README.md

```diff
@@ -231,9 +240,9 @@ class ContextsActivatedTest {
 ##### Adding mock beans
 
 Sometimes you might need to add a mock for a bean that cannot be part of the test deployment, e.g. the original bean implementation has dependencies which cannot be satisfied in the test environment.
-Very often, it's an ideal use case for mocking libraries, ie. to create a bean instance with the desired behavior.
+Very often, it's an ideal use case for mocking libraries, i.e. to create a bean instance with the desired behavior.
 In this case, there are two options.
-The first option is to add a [producer method](http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#producer_method) to the test class and add the test class to the deployment.
+The first option is to add a [producer method](https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#producer_method) or [field](https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#producer_field) to the test class and add the test class to the deployment.
```

Comment:
Ah, right, I just had seen a mix of 1.2 and 2.0 URLs and all with HTTP, so I thought I make them consistent 2.0 with HTTPS.
I didn't even thinka bout 3.0 being correct.
Maybe because it is weld-junit 3.x and weld 3.x is still CDI 2, while the target of course is weld 4.x.
However, fixed now.